packages/website/ts/blockchain.ts

```diff
@@ -543,6 +554,13 @@ export class Blockchain {
 
         this._dispatcher.updateBlockchainIsLoaded(true);
     }
+    public async getFirstAccountIfExistsAsync() {
```

Comment:
the 'IfExists' portion of the method name suggests that `undefined` will be returned if we don't find an address